packages/EasyErrorHandler/src/Bridge/Symfony/Resources/config/services.php

```diff
@@ -26,7 +30,8 @@
     $services = $containerConfigurator->services();
     $services->defaults()
         ->autowire()
-        ->autoconfigure();
+        ->autoconfigure()
+        ->bind('string $errorCodesInterface', ErrorCodesInterface::class);
```

Comment:
I'm not 100% sure what's the objective with this and the default ErrorCodesProvider but `bind` is working only within the context of the current file which means apps won't be able to override it from their own config files, and it feels strange to create an empty interface to use reflection on it.

Could you please clarify what is the intention with this default error codes provider?

- For apps to use it by changing the class passed to its constructor? Then I think we shouldn't have an empty interface but have a nullable config to allow apps to define which class to use, and if it's null then the provider simply returns an empty array.
- For apps to replace the error codes provider with their own? Then let's simply return an empty array from the default one and it's done.

Reply:
The first option, i.e. to allow apps to define which class to use as error codes source. Yes, we can use nullable `$errorCodesInterface` to get rid of this empty `ErrorCodesInterface`.

Reply:
@akl-95 ok then I think we need to introduce a config for it

Reply:
Reworked.